source/publishing_data/04_configuring_a_source/connectors/json.rst

```diff
@@ -0,0 +1,322 @@
+If ``content.trains`` is set as JSON root, the resulting dataset will be:
+
++-----+------------------------------------------------------------------------------+
+| id  | info                                                                         |
++-----+------------------------------------------------------------------------------+
+| 123 | {"origin": "Agra Cantt", "destination": "New Delhi", "name": "Agra Express"} |
++-----+------------------------------------------------------------------------------+
+| 555 | {"origin": "Balurghat", "destination": "Sealdah", "name": "Gour Express"}    |
++-----+------------------------------------------------------------------------------+
+
+To only extract the ``info`` JSON objects, and skip the ``id`` number, the correct JSON object is ``info``.
+If ``info`` is set as JSON object, the resulting dataset will be:
+
++--------------+------------+-------------+
+| name         | origin     | destination |
++--------------+------------+-------------+
+| Agra Express | Agra Cantt | New Delhi   |
++--------------+------------+-------------+
+| Gour Express | Balurghat  | Sealdah     |
++--------------+------------+-------------+
+
+
+JSON Line
+---------
+
+This connector expects a file in which each line is a one-line JSON documents. The whole file is not a valid JSON document but each line is.
+
+The connector supports 2 modes:
+
+    * one JSON array on each line (or one array altogether)
+    * one JSON object on each line
+
+Supported field types
+~~~~~~~~~~~~~~~~~~~~~
+
+* Regular fields (decimal, string)
+* Array: will be used as-is
+* JSON object: will be used as-is
+
+Creation
+~~~~~~~~
+
+See :doc:`how to source a file</publishing_data/01_creating_a_dataset/sourcing_data>`.
+
+Configuration
+~~~~~~~~~~~~~
+
+.. list-table::
+   :header-rows: 1
+
+   * * Name
+     * Description
+     * Usage
+   * * Extract filename
+     * Creates a new column at the end of the dataset with the name of the source file.
+     * By default, the box is not checked. Check the box to extract filename in an added column.
+   * * First line number
+     * For files which do not start at the very first line, it is possible to decide which line is to be considered the first one. The lines above will be skipped from the dataset.
+     * By default, the dataset starts at line 1. Indicate the number of the line that should be considered the beginning of the dataset.
+   * * Headers
+     * For files which first line contains column titles.
+     * By default, the box is checked. It makes the values of the first line field labels. Uncheck the box if the first line doesn't contain titles but data: the field labels will then be empty by default.
+
+
+
+JSON Dict
+---------
+
+.. admonition:: Important
+   :class: important
+
+   This connector is not available by default. Please contact OpenDataSoft for the activation of this connector on a given OpenDataSoft domain.
```

Comment:
Yes it is!

Reply:
But if it is, why is is listed among the "Additional data processing components" (License > Information, in the back office of my portal)? Shouldn't there only be extractors/harvesters/processors activated on demand listed in there?